SYCL/ESIMD/api/functional/operators/operator_decrement_and_increment_accuracy_core.cpp

```diff
@@ -0,0 +1,45 @@
+//==------- operator_logical_not.cpp  - DPC++ ESIMD on-device test ---------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// Test for simd increment and decrement operators.
+// The test creates source simd instance and call increment or decrement
+// operator opera with reference data. The test verifies that in the output data
+// contained correctness data according to chosen operator and has no precision
+// differences with interaction with floating point data types.
+
+#include "operator_decrement_and_increment.hpp"
+
+using namespace esimd_test::api::functional;
+
+int main(int, char **) {
+  sycl::queue queue(esimd_test::ESIMDSelector{},
+                    esimd_test::createExceptionHandler());
+
+  bool passed = true;
+
+  const auto u_types = get_tested_types<tested_types::fp>();
```

Comment:
It's a bit unclear for me what does `u` in `u_types` mean
```suggestion
  const auto fp_types = get_tested_types<tested_types::fp>();
```

Reply:
this comment applied [here](https://github.com/intel/llvm-test-suite/pull/827/commits/7ff74964cc8f28e39cce66778060f321038d13cc#diff-009d4498abf01fa40a27db546175221887f54a04b58f5db8e4eb7fbbf1a1af1fL33-R33)